### DISCUSSION TITLE
Unverfied Drives

### DISCUSSION THREAD

**Discussion by HPMSGEN8** [007revad/Synology_HDD_db#142] [Q&A] [answered]
Good Day

Does this fix the Unverfied Drives issue on DSM v7.2 Update 3

Thanks

**Comment by 007revad** [comment]
Yes. 

But are you saying your drives didn't show as unverified until you updated to DSM 7.2 Update 3?

**Reply by HPMSGEN8** [reply]
Sorry they are all showing Unverified. - I upgraded my setup for 7.1 to 7.2 Update 3

WD60EFRX x 3 - SATA
Seagate 16TB Exos SAS x1

**Reply by HPMSGEN8** [reply]
DS3622xs+

2x WD60EFRX-68MYMN1
1x WD60EFRX-68L0BN1
1x Seagate ST16000NM004G

**Reply by HPMSGEN8** [reply]
@007revad Let me know if you need any more info from my side. Thanks again

**Reply by 007revad** [reply]
It's strange that they didn't show as unverified on a DS3622xs+ in older DSM 7.2 versions.

Yes, this script will tell DSM that your drives are compatible so you won't get the unverified warnings.

**Reply by HPMSGEN8** [reply]
What is the cause have you tested on Update 3 at all? 

I really am lost with this.

Thanks

**Reply by 007revad** [reply, marked as answer]
Update 3 did cause some issues that I had to solve.

Though for some people the script doesn't work until they "run the script and reboot" 2 or 3 times. They never said if they had the script scheduled to run at shutdown, or bootup, or ran it manually. Though 1 person did say that they had it scheduled to run at shutdown **and** bootup.

**Reply by HPMSGEN8** [reply]
Thanks a lot will check this out and see. Thanks again
